### PR TITLE
French locale

### PR DESCRIPTION
**Pull request by ghost**
Creating a French locale. Should be reviewed by Julien aka Sphinx regarding tech language.

### CODE REVIEW

**Comment by SebastianZ** [comment]
Thanks for the translations, @goofy-bz!

@SphinxKnight, could you please do the review?

Sebastian

File: locale/fr.properties [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+empty_brackets=Parenthèses vides
+
+# Description of test checking for empty brackets
+empty_brackets_desc=Les macros qui n’ont pas de paramètres comme {{foo()}} n’ont pas besoin de parenthèses de listes de paramètres et devraient donc être supprimées.
+
+# Name of test checking for ’style’ HTML attributes
+style_attributes=Attributs de style
+
+# Description of test checking for ’style’ HTML attributes
+style_attributes_desc=Les éléments HTML ne devraient pas inclure d’attributs de « style » mais utiliser plutôt des classes. Ils devraient donc être supprimés quand c’est possible.
```
Comment (SphinxKnight) [inline locale/fr.properties]: "attributs de « style »" should be "attributs « style »"

File: locale/fr.properties [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+# Name of test checking for ’name’ HTML attributes
+name_attributes=Attributs de nom.
```
Comment (SphinxKnight) [inline locale/fr.properties]: "Attributs « name »" instead

File: locale/fr.properties [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+wrong_highlighted_line_desc=Un bloc de code comprend une ligne surlignée, en-dehors de la zone des lignes de code.
+
+# Name of test checking the order and names of API syntax headlines
+api_syntax_headlines=Syntaxe des sous-titres pour les API
+
+# Description of test checking the order and names of API syntax headlines
+api_syntax_headlines_desc=La syntaxe de sous-titres pour les API doit être, dans cet ordre ’Paramètres’, ’Valeur retournée’ et ’Exceptions’, et non pas ’Retours’, ’Erreurs’ ou ’Erreurs retournées’.
+
+# Name of test checking whether there are <code> tags within <pre> blocks
+code_in_pre=<code> en <pre>
```
Comment (SphinxKnight) [inline locale/fr.properties]: Élément <code> dans <pre>

**Comment by SphinxKnight** [comment]
@goofy-bz some minor typos :)

**Comment by ghost** [comment]
Thanks Julien, corrections done. Please Florian tell me if everything is ok because I am rather awkward with github process.

**Comment by Elchi3** [comment]
Looks good to me. Thank you Goofy and Sphinx! A best practice is to use one commit that includes subsequent corrections, but this is fine with me as is. If you are curious, you can read more about that by searching for "git squash commits" in your search engine.

**Pull request merged**